src/interactive/scala/tools/nsc/interactive/Pickler.scala

```diff
@@ -33,8 +33,8 @@ import java.io.Writer
  *  These Picklers build on the work of Andrew Kennedy. They are most closely inspired by
  *  Iulian Dragos' picklers for Scala to XML. See:
  *
- *  <a href="http://code.google.com/p/gdata-scala-client/wiki/DevelopersGuide">
- *  http://code.google.com/p/gdata-scala-client/wiki/DevelopersGuide
+ *  <a href="https://code.google.com/p/gdata-scala-client/wiki/DevelopersGuide">
+ *  https://code.google.com/p/gdata-scala-client/wiki/DevelopersGuide
  *  </a>
```

Comment:
I am not familiar with Scala, but could this link be changed like this?
```suggestion
 * [[https://code.google.com/p/gdata-scala-client/wiki/DevelopersGuide]]
```